runtime/doc/syntax.txt

```diff
@@ -4816,16 +4810,20 @@ is mostly used, because it looks better.
 ==============================================================================
 13. Highlight command			*:highlight* *:hi* *E28* *E411* *E415*
 
-There are two types of highlight groups:
-- The built-in |highlight-groups|.
-- The ones used for specific languages.  For these the name starts with the
-  name of the language.  Many of these don't have any attributes, but are
-  linked to a group of the second type.
-							*hitest.vim*
-You can see all the groups currently active with this command: >
-    :so $VIMRUNTIME/syntax/hitest.vim
-This will open a new window containing all highlight group names, displayed
-in their own color.
+Vim uses a range of highlight groups which fall into two categories: Editor
```

Comment:
`Nvim` :)

Reply:
But some of following lists are not in vim and we have changed a lot of places to use `Nvim` 🤔